kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/utils/KopTopic.java

```diff
@@ -103,4 +106,10 @@ public static String toString(String topic, int partition, String namespacePrefi
         return (new KopTopic(topic, namespacePrefix)).getPartitionName(partition);
     }
 
+    public static boolean isInternalTopic(final String fullTopicName) {
+        String partitionedTopicName = TopicName.get(fullTopicName).getPartitionedTopicName();
+        return partitionedTopicName.endsWith("/" + GROUP_METADATA_TOPIC_NAME)
+                || partitionedTopicName.endsWith("/" + TRANSACTION_STATE_TOPIC_NAME);
+    }
```

Comment:
`KafkaRequestHandler#isInternalTopic` is still used in `handleTopicMetadataRequest`, please reuse it.

Reply:
Sure. Addressed.